internal/cli/tiltfile_result.go

```diff
@@ -0,0 +1,87 @@
+package cli
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"os"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	tiltanalytics "github.com/windmilleng/tilt/internal/analytics"
+	"github.com/windmilleng/tilt/internal/tiltfile"
+	"github.com/windmilleng/tilt/pkg/logger"
+	"github.com/windmilleng/tilt/pkg/model"
+)
+
+const TiltfileErrExitCode = 5
```

Comment:
it feels bikesheddy to ask "why 5?" but it also feels like just putting 5 here is making it really tempting to ask - maybe a "// arbitrary non-1 value chosen to allow callers to distinguish between tilt errors and tiltfile errors?"